openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -192,3 +192,67 @@ def get_children_count(self, obj):
         Returns the number of child tags of the given tag.
         """
         return len(obj.sub_tags)
+
+
+class TaxonomyTagCreateBodySerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer of the body for the Taxonomy Tags CREATE view
```

Comment:
```suggestion
    Serializer of the body for the Taxonomy Tags CREATE request
```
If you say "view", it's unclear if this is for the request body or the response body.

Reply:
Got it, updated.